Classes/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -648,6 +648,8 @@ public function highlight($fragmentSize, $fragmentCount = null)
      */
     public function query(NodeInterface $contextNode)
     {
+        $this->elasticSearchClient->setDimensions($contextNode->getContext()->getTargetDimensions());
```

Comment:
Maybe we can have a settings or at least a method to query the current specific indexe (current dimensions) or the global index (all dimensions). Some client can found useful to have a query in a foreign language that return results if the document is also available in the current dimensions.